src/Access/TaxonomyDeleteAccessCheck.php

```diff
@@ -0,0 +1,256 @@
+       * it is OK to delete it.
+       */
+      if ($hasAccessControlMembers || $assigned_content) {
+
+        $override_allowed = $this->account->hasPermission('allow taxonomy term delete');
+
+        if ($assigned_content && !$override_allowed) {
+          $this->messenger->addWarning(t("The term %term is being used to tag content and may not be deleted.",
+            ['%term' => $term->getName()]));
+          $retval = FALSE;
+        }
+        elseif ($assigned_content) {
+          $this->messenger->addWarning(t("The term %term is being used to tag content.",
+            ['%term' => $term->getName()]));
+          $retval = TRUE;
+        }
+
+        if ($hasAccessControlMembers) {
+          $this->messenger->addWarning(t("The term %term is being used for access control and may not be deleted.",
+           ['%term' => $term->getName()]));
+          $retval = FALSE;
+        }
+
+      }
+    }
+
+    return $retval;
+
+  }
+
+  /**
+   * Determines if this term has active members in it.
+   *
+   * @return bool
+   *   TRUE if the term has members, FALSE otherwise.
+   */
+  private function doesTermHaveMembers(TermInterface $term) {
+
+    /** @var array $sections */
+    $sections = $this->getActiveSections($term);
+
+    if (count($sections) > 0) {
+      return TRUE;
+    }
+
+    return FALSE;
+  }
+
+  /**
+   * Helper function to extract the entity for the supplied route.
+   *
+   * @return null|TermInterface
+   *   A term interface object if the term exists. NULL otherwise.
+   */
+  private function getRouteEntity() {
+    $route_match = $this->route;
+    // Entity will be found in the route parameters.
+    if (($route = $route_match->getRouteObject()) && ($parameters = $route->getOption('parameters'))) {
+      // Determine if the current route represents an entity.
+      foreach ($parameters as $name => $options) {
+        if (isset($options['type']) && strpos($options['type'], 'entity:') === 0) {
+          $entity = $route_match->getParameter($name);
+          if ($entity instanceof TermInterface) {
+            return $entity;
+          }
+
+          // Since entity was found, no need to iterate further.
+          return NULL;
+        }
+      }
+    }
+  }
+
+  /**
+   * Inspect the given taxonomy term.
+   *
+   * This will determine if there are any active users assigned to it.
+   *
+   * @param \Drupal\taxonomy\TermInterface $term
+   *   The Taxonomy Term to inspect.
+   *
+   * @return array
+   *   An array of the users assigned to this section.
+   */
+  private function getActiveSections(TermInterface $term) {
+    /** @var \Drupal\workbench_access\UserSectionStorageInterface $sectionStorage */
+    $sectionStorage = $this->userSectionStorage;
+
+    /** @var \Drupal\Core\Config\Entity\ConfigEntityStorage $scheme */
+    $scheme = $this->entityTypeManager->getStorage('access_scheme');
+
+    $access = $scheme->load("access_section");
```

Comment:
this is the issue @jasonpartyka you're hardcoding the access scheme ID, which I guess is what your site uses, but isn't what the test uses

Reply:
I figure you'll probably have to do something like this:

```php
return array_reduce($this->entityTypeManager->getStorage('access_scheme')->loadMultiple(), function (array $editors, \Drupal\workbench_access\Entity\AccessSchemeInterface $scheme) use ($sectionStorage, $term) {
  $editors += $sectionStorage->getEditors($scheme, $term->id());
  return $editors;
}, []);
```

Reply:
@jasonpartyka Doh, It's from my 'vanilla' install (instead of my company's site) -- I was being too generic!

Reply:
Ok, I corrected this and it's pushed up.